<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Ashu" failures="1" tests="4" name="Test" time="16.78" errors="0" timestamp="2025-01-26T12:57:11 IST">
  <testcase classname="com.testCases.AutomationBlogSpot" name="firstTestCase" time="6.267"/>
  <testcase classname="com.testCases.PaginationTests" name="checkProduct" time="10.867"/>
  <testcase classname="com.testCases.AutomationBlogSpot" name="firstTestCase" time="5.13"/>
  <testcase classname="com.testCases.PaginationTests" name="getPriceofProduct" time="5.277">
    <failure type="java.lang.AssertionError" message="expected [$2.998] but found [$2.99]">
      <![CDATA[java.lang.AssertionError: expected [$2.998] but found [$2.99]
at com.testCases.PaginationTests.getPriceofProduct(PaginationTests.java:19)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- getPriceofProduct -->
</testsuite> <!-- Test -->
